DashWallet.xcodeproj/project.pbxproj

```diff
@@ -11701,7 +11701,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = dashwallet/dashwallet.entitlements;
-				CURRENT_PROJECT_VERSION = 1;
+				CURRENT_PROJECT_VERSION = 2;
```

Comment:
it's better to keep always 1, appstoreconnect increases build number while uploading